packages/x-license-pro/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license-pro",
-  "version": "5.17.0",
+  "version": "5.17.1",
```

Comment:
Technically the version should be aligned with other packages (data-grid)/root version.
Based on previous releases of this package—it seems that we did follow this pattern.
```suggestion
  "version": "5.17.11",
```